src/docs/goose-book/src/config/defaults.md

```diff
@@ -79,4 +79,4 @@ For example, without any run-time options the following load test would automati
     Ok(())
 ```
 
-Find a complete list of all configuration options that can be configured with custom defaults [in the developer documentation](https://docs.rs/goose/*/goose/config/enum.GooseDefault.html), as well as complete details on [how to configure defaults](https://docs.rs/goose/*/goose/config/trait.GooseDefaultType.html).
\ No newline at end of file
+Find a complete list of all configuration options that can be configured with custom defaults [in the developer documentation](https://docs.rs/goose/*/goose/config/enum.GooseDefault.html), as well as complete details on [how to configure defaults](https://docs.rs/goose/*/goose/config/trait.GooseDefaultType.html).
```

Comment:
What's changed here? I'm not seeing a difference in the Github UI?

Reply:
That's confusing me.